_projects/2022/sunpy/scraper.md

```diff
@@ -0,0 +1,60 @@
+---
+name: Scraper rewrite
+desc: Improve the current scraper to download data
+requirements:
+ - Familiarity with Python
+ - Knowledge of Python regex
+ - Familiar with web protocols
+difficulty: medium
+issues:
+ - https://github.com/sunpy/sunpy/issues/4888
+mentors:
+ - nabobalis
+ - samaloney
+ - hayesla
+ - dpshelio
+initiatives:
+ - GSOC
+project_size:
+ - medium
+ - large
```

Comment:
I was unable to decide.

Reply:
I think this could probably be a medium-sized project - what do you think?

Reply:
Yeah, I think I will pick this to be medium.